backend/src/entities/UserEntity.ts

```diff
@@ -0,0 +1,23 @@
+import {
+  Column,
+  Entity,
+  PrimaryGeneratedColumn,
+} from 'typeorm';
+
+@Entity()
+export class UserEntity {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  name: string;
+
+  @Column()
+  username: string;
+
+  @Column()
+  email: string;
```

Comment:
Either email or Username might have to be unique?
Prob check with @kehwhy @cdjamalov